lib/core/engine.js

```diff
@@ -194,6 +195,16 @@ class Engine {
     });
   }
 
+  swarmService(_options) {
+    this.registerModule('swarm', {
+      addCheck: this.servicesMonitor.addCheck.bind(this.servicesMonitor),
+      storageConfig: this.config.storageConfig,
+      host: _options.host,
```

Comment:
is _options.host and _options.port coming from the cmd line, if so, they are not specified, does commander automatically include them in the options? regardless we should add those parameters in cmds.js

Reply:
Actually these are unused, and have been removed now. The host and port (used later in the swarm module) come from the storage config. Are you sure you want me to add these as command line options?